new.sh

```diff
@@ -205,7 +205,7 @@ cat > "${root}/bin/configs/${gen_name_camel}-${gen_type}-petstore-new.yaml"<<EOF
 generatorName: ${gen_name_camel}
 outputDir: samples/${gen_type}/petstore/${gen_name_camel_path}
 inputSpec: modules/openapi-generator/src/test/resources/3_0/petstore.yaml
-templateDir: modules/openapi-generator/src/main/resources/${gen_name_camel}
+templateDir: modules/openapi-generator/src/main/resources/${gen_name_camel}-${gen_type}
```

Comment:
@AndersSpringborg thanks for the PR but I think we usually use the generator name directly as the template folder name (e.g. lua, perl) without adding the generator type (e.g. client, server)

Reply:
Oh yeah, i can see what you mean now!

If you follow the instruction on how to make a new generator, it gives an error, that I cannot find that template. I'll look into what courses the problem tomorrow 😊

Reply:
In the the script, the template folder is created with `${gen_name_camel}-${gen_type}`
https://github.com/OpenAPITools/openapi-generator/blob/0f5e7d1e3cb9642bd46ec89ffe0bf66a922029f2/new.sh#L193-L200

So the template dir in the config, should have the same path
https://github.com/OpenAPITools/openapi-generator/blob/0f5e7d1e3cb9642bd46ec89ffe0bf66a922029f2/new.sh#L208

Reply:
Good point. What about updating these lines instead following the convention that I mentioned?
```
# Step 3: Create resource files 
 mkdir -p "${root}/modules/openapi-generator/src/main/resources/${gen_name_camel}-${gen_type}" 
 echo "Creating modules/openapi-generator/src/main/resources/${gen_name_camel}-${gen_type}/README.mustache" && \ 
     touch "${root}/modules/openapi-generator/src/main/resources/${gen_name_camel}-${gen_type}/README.mustache" 
 echo "Creating modules/openapi-generator/src/main/resources/${gen_name_camel}-${gen_type}/model.mustache" && \ 
     touch "${root}/modules/openapi-generator/src/main/resources/${gen_name_camel}-${gen_type}/model.mustache" 
 echo "Creating modules/openapi-generator/src/main/resources/${gen_name_camel}-${gen_type}/api.mustache" && \ 
     touch "${root}/modules/openapi-generator/src/main/resources/${gen_name_camel}-${gen_type}/api.mustache" 
```

Reply:
Let me know if that's how you want the script to work! I'll go through it, and make sure it works with this naming convention

Reply:
Please update based on https://github.com/OpenAPITools/openapi-generator/pull/9321#discussion_r619915054. Thank you.